### ISSUE TITLE
Can I emulate the action of clicking on the icon for "Add-ons"? Or, say, to have an extension run through a shortcut?

### ISSUE THREAD

**Issue by llinfeng** [vimperator/vimperator-labs#812]
##### Issue type:
- Feature request

##### Version:

```
Vimperator:	3.16.0.20170928091306 (created: 2017/09/28 09:13:06)
Firefox:	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:55.0) Gecko/20100101 Firefox/55.0
```

##### Description:
Can I use `vimperator` to emulate the clicking of Add-on icons that are parked on the Toolbar? (The banner that has URL, and a set of icons. This is revealed by `set gui+=navigation`.)

In particular, I need to click the icon for "Zotero Connector", which is the "folder-icon" that I kept as the right-most icon, as follows. This serves the purpose of adding an "item" to the Zotero collection, which I would repeat multiple times (20+) through out the day.
![folder-icon](https://user-images.githubusercontent.com/5890407/32803703-6a304426-c952-11e7-9496-af023a6cc638.png)

* Status of Zotero: so far, Zotero is of version 5.0, and there is no longer a stand-alone Firefox Add-on available. Thus, the community is providing a "Zotero Connector", which is only clickable. Such "Connector" add-on does not have any Menu entry, thus cannot be evoked by `:emenu XXX`; nor, does it have a Java Script port. Typing `:js Zotero` does not yield any valid function to call.
* In short, this Add-on is only working when one click on it.

##### Expected behavior:
* Either we can emulate the clicking behavior of those Toolbar icons; 
* Or, since the "Zotero Connector" is an "Extension", should there be a way to assign a shortcut/mapping to trigger it? 
* More intensively, since Zotero is opening up the API interface, as [documented](https://www.zotero.org/support/dev/web_api/v3/start), a function that directly update entries on the Zotero server may also be a solution.

##### Steps to reproduce:
Anyone who try to install Zotero 5.0 and Zotero Connector shall find that there is no shortcuts provided by Zotero. 
* Zotero link: https://www.zotero.org/download/

**Comment by llinfeng** [comment]
The problem is mediated by finding out that there is a secrete shortcut that helps saving items to Zotero (V 5.0): `Ctrl + Shift + S`. Yet, this key is not yet documented anywhere: not in list of shortcuts in the setting of the Standalone Zotero, nor the Connector.

**Comment by llinfeng** [comment]
The problem is mediated by finding out that there is a secrete shortcut that helps saving items to Zotero (V 5.0): `Ctrl + Shift + S`. Yet, this key is not yet documented anywhere: not in list of shortcuts in the setting of the Standalone Zotero, nor the Connector.
